instrumentation/logback/logback-appender-1.0/library/src/main/java/io/opentelemetry/instrumentation/logback/appender/v1_0/internal/LoggingEventMapper.java

```diff
@@ -46,18 +47,21 @@ public final class LoggingEventMapper {
   private final boolean captureCodeAttributes;
   private final boolean captureMarkerAttribute;
   private final boolean captureKeyValuePairAttributes;
+  private final boolean captureLoggerContext;
 
   public LoggingEventMapper(
       boolean captureExperimentalAttributes,
       List<String> captureMdcAttributes,
       boolean captureCodeAttributes,
       boolean captureMarkerAttribute,
-      boolean captureKeyValuePairAttributes) {
+      boolean captureKeyValuePairAttributes,
+      boolean captureLoggerContext) {
```

Comment:
This is a pretty sizable argument count -- and all those consecutive booleans are pretty prone to error. If nothing else, it's impossible for a reader to really grok.  A builder could help. 🤷🏻

Reply:
created https://github.com/open-telemetry/opentelemetry-java-instrumentation/issues/9889